yang-models/dell-base-if-vlan.yang

```diff
@@ -6,21 +6,22 @@ module dell-base-if-vlan {
     import ietf-interfaces {
         prefix "if";
     }
-    import ietf-yang-types {
-        prefix "yang";
-    }
-	
     import dell-base-common {
         prefix "base-cmn";
     }
 
+    import iana-if-type { prefix "ianaift"; }
+
     organization "Dell EMC";
     contact "http://www.dell.com/support";
 
     description "This model augments the standard interface and adds Dell EMC specific
     				vlan extensions.";
+    revision "2018-09-21" {
```

Comment:
For every revision statement,  we have standardized the inclusion of a reference statement. See the example below.

 revision "2018-08-10" {
        description "Fixing pyang IETF errors and adding documentation to the model.";
        reference "Network Platform Abstraction";
    }